### DISCUSSION TITLE
going nuts

### DISCUSSION THREAD

**Discussion by reese555** [vladmandic/sdnext#1477] [General]
this driving me nuts. ive tried everything i could think of,rolled back video drivers,reinstalled vlad even did a windows reset. the card is fine it plays everything runs cinema 4d fine its just vlad. dunno what else to try

Exception in thread MemMon:
20:45:03-610074 ERROR    Exception: CUDA error: an illegal memory access was encountered
                         CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace
                         below might be incorrect.
                         For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
                         Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.
![image](https://github.com/vladmandic/automatic/assets/2958556/bb914e0c-1e98-4677-adbb-b48974a73f1a)

**Comment by vladmandic** [comment]
basically from the screenshot above, cuda does not have access to gpu memory at all.
first thing i'd try is to do a full nvidia driver reinstall. which driver version are you using?

**Reply by reese555** [reply]
i was using very latest but saw comments about it causing issues so i rolled back 531.29 and problem still there ive seen a few people post error like ive been getting. i thought going back would fix it. would cuda toolkit be an issues? on latest version of that too

**Reply by vladmandic** [reply]
what do you mean "on latest version"? app installer automatically installs torch which includes cuda it needs, you should not have cuda manually installed.
